src/Makefile.am

```diff
@@ -47,6 +49,7 @@ EXTRA_LIBRARIES = \
   libbitcoin_common.a \
   libbitcoin_server.a \
   libbitcoin_cli.a \
+  libsnark.a \
```

Comment:
In ``*_LDADD`` libsnark is after libzcash; is it important that this is different? If not then make it the same.